Sources/Linq2DynamoDb.DataContext.Tests/EntityManagementTests/Versioning/EntityVersioningTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Linq2DynamoDb.DataContext.Tests.Entities;
+using Linq2DynamoDb.DataContext.Tests.Helpers;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace Linq2DynamoDb.DataContext.Tests.EntityManagementTests.Versioning 
+{
+    [TestFixture]
+    class EntityVersioningTests : DataContextTestBase
```

Comment:
maybe add a test case for the scenario where you submit and THEN read and modify the entity again?
(in other words, the non-exceptional case. verify you can make 2 updates on a single entity and verify both changes as you make them).